auth/src/args.rs

```diff
@@ -31,7 +31,7 @@ pub struct StartArgs {
 
     /// Auth JWT signing private key, as a base64 encoding of
     /// a PEM encoded PKCS#8 v1 formatted unencrypted private key.
-    #[arg(long, default_value = "")]
+    #[arg(long)]
```

Comment:
No particular reason, I wanted to force the program to fail early in case the JWT signing key is not (provided or valid). Removing the `default_value` here is the same as making the arg as `required` and simplifies the code. It is a must to provide a value and if the value is invalid (base64 of a PEM encoded private key), we fail early.

Reply:
More context here: https://github.com/shuttle-hq/shuttle/pull/1402#discussion_r1401851215.